projects/packages/codesniffer/Jetpack/Sniffs/Functions/SetCookieSniff.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
```suggestion
<?php //phpcs:ignore WordPress.Files.FileName.NotHyphenatedLowercase
```
PHPCS in VScode complained about this.